src/Process.php

```diff
@@ -120,6 +126,67 @@ public function start(LoopInterface $loop, $interval = 0.1)
     }
 
     /**
+     * This is a special implementation of the start method for the Windows operating system.
+     * This is needed because windows has a broken implementation of stdout / stderr pipes that
+     * cannot be fixed by PHP.
+     *
+     * @param LoopInterface $loop        Loop interface for stream construction
+     * @param float         $interval    Interval to periodically monitor process state (seconds)
+     * @throws RuntimeException If the process is already running or fails to start
+     */
+    protected function startWindows(LoopInterface $loop, $interval = 0.1)
+    {
+        $cmd = $this->cmd;
+
+        $stdoutName = tempnam(sys_get_temp_dir(), "out");
+        $stderrName = tempnam(sys_get_temp_dir(), "err");
```

Comment:
Perhaps (somehow) expose an option to explicitly define the path to avoid privacy issues on multi user systems?